Neos.ContentGraph.DoctrineDbalAdapter/src/Domain/Repository/ContentSubgraph.php

```diff
@@ -355,27 +355,34 @@ public function countAncestorNodes(NodeAggregateId $entryNodeAggregateId, CountA
 
     public function findClosestNode(NodeAggregateId $entryNodeAggregateId, FindClosestNodeFilter $filter): ?Node
```

Comment:
maybe the adjustments here will improve https://github.com/neos/neos-development-collection/issues/4764 already?